templates/pharmacies/general_edit.html

```diff
@@ -45,7 +45,18 @@
 		<input TYPE="TEXT" NAME="fax" SIZE="12" VALUE="{$pharmacy->get_fax()|escape:'html'}" onKeyDown="PreventIt(event)" />
 	</td>
 </tr>
-
+<tr>
+    <td VALIGN="MIDDLE" >{xl t='NPI'}</td>
+    	<td VALIGN="MIDDLE" >
+		<input TYPE="TEXT" NAME="npi" SIZE="12" VALUE="{$pharmacy->get_npi()}" onKeyDown="PreventIt(event)" />
```

Comment:
note this form now does html escaping (yes, it's another new thing :) )
``````
<input TYPE="TEXT" NAME="npi" SIZE="12" VALUE="{$pharmacy->get_npi()|escape:'html'}" onKeyDown="PreventIt(event)" />